app/assets/javascripts/app/views/location_view.js

```diff
@@ -10,7 +10,7 @@ app.views.Location = Backbone.View.extend({
   },
 
   render: function(){
-    $(this.el).append('<img alt="ajax-loader" src="'+ImagePaths.get('ajax-loader.gif')+'">');
+    $(this.el).append("<img alt='ajax-loader' src='" + ImagePaths.get('ajax-loader2.gif') + "'>");
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'ImagePaths' is not defined.